dependencies.yaml

```diff
@@ -2,6 +2,7 @@ options:
   buildHeader: [ "load(\"@io_bazel_rules_scala//scala:scala_import.bzl\", \"scala_import\")" ]
   languages: [ "java", "scala:2.11.8" ]
   resolverType: "coursier"
+  strictDeps: true
   resolvers:
```

Comment:
You use strictDeps here, but below strictVisibility. I think I like the latter better since strictDeps has a meaning already in bazel but this it different.

Reply:
Also, let’s leave this out here since this is the default I believe.

Reply:
Good spot. `strictVisibility` is the flag. I'll re-verify and fix it after I split the pr's. I'll add `strictVisibility: true` as well.